airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1944,8 +1944,10 @@ components:
             $ref: "#/components/schemas/Notification"
         firstCompletedSync:
           type: boolean
+          nullable: true
```

Comment:
what are you trying to achieve here? because these fields are not in the `required` list above they should already be nullable and should not require the extra `nullable` field.

Reply:
When I called the workspace endpoint with the generated API client it raises an error during serialization of the response because it expected this field to be not nullable. When I make this change in `config.yaml` the problem is bypassed. I'm going to check if I can find another solution by tweaking some generator options.

Reply:
yeah. that seems like kinda big deal. swagger generation is always a little jenky around the edges, but nullable fields is a pretty core feature, so it seems like a really bad sign if the generator can't handle it because it means there will be other problems.

one thing to test is instead of using the `nullable` field if instead setting `type: [boolean, null]` works. I don't like this as much as the pattern we already use but if we needed to switch to this pattern it may not be the end of the world.

Reply:
I found this issue https://github.com/OpenAPITools/openapi-generator/issues/4816 on OpenApiTools repo, this is for their C# generator but it looks like the exact same problem I have with the Python one. I'll continue to investigate and try  `type: [boolean, null]`.  Let me know if you think that I should rather implement the client myself.

Reply:
we definitely don't want to implement it ourselves. my guess is the C# bug is language specific (since C# is actually a typed language)

Reply:
I tried `[boolean, null]` but this is not a valid spec according to open-api-generator.
I think I found a workaround. I can disable the type validation of the response. It also disables deserialization to a model and leads us to manipulate the raw response as a python dict. It's not the most elegant solution but I prefer this rather than editing the API spec. Related commit: https://github.com/airbytehq/airbyte/pull/9277/commits/a586719d83f229c55bb2e727b9320d1aa1184f59
I also reverted my changes on `airbyte-api`'s `config.yaml`.